contracts/util/LoanLib.sol

```diff
@@ -0,0 +1,339 @@
+pragma solidity 0.5.17;
+pragma experimental ABIEncoderV2;
+
+// Libraries
+import "@openzeppelin/contracts-ethereum-package/contracts/math/SafeMath.sol";
+import "../util/TellerCommon.sol";
+import "../util/AddressLib.sol";
+
+// Interfaces
+import "../interfaces/SettingsInterface.sol";
+import "../interfaces/EscrowInterface.sol";
+import "../interfaces/LoansInterface.sol";
+
+// Contracts
+
+library LoanLib {
+    using SafeMath for uint256;
+    using NumbersLib for uint256;
+    using NumbersLib for int256;
```

Comment:
I think NumbersLib is defined for uint256 (not int256).

Reply:
Some functions can be used for both